tools/bigwig_outlier_bed/.shed.yml

```
@@ -0,0 +1,20 @@
+categories:
+- Sequence Analysis
+description: pybigtools and numpy code to find continuous runs above a high or below a low quantile cutpoint in bigwig files
+homepage_url: https://github.com/fubar2/bigwig_outlier_bed
+long_description: |
+  Bigwigs are great. Peaks are easy to see. Small runs of very low values not so much
+
+  Using numpy to segment bigwig values into regions of contiguous high and low values, this tool writes bed files
+  containing those regions, with a score set to 1 or -1 depending on whether above the top quantile or below the
+  low quantile.
+
+  Quantile values recommended are 0.01 and 0.99. They are calculated for each chromosome and vary a bit.
+  Ideally should be estimated over the entire assembly but not feasible without sampling due to RAM hoggery.
+  Minimum window size of 10 will give a very, very low risk of random false positives at about 0.01**10 for 0.01
+  quantile cutoff for example.
+
+name: bigwig_outlier_bed
+owner: iuc
+remote_repository_url: https://github.com/jackh726/bigtools
```

```suggestion
remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigwig_outlier_bed
```

This should point to the repo of the wrapper.